packages/conf/src/index.ts

```diff
@@ -278,7 +278,7 @@ export function fallbackLanguageMigration(
 
   config.locales.forEach((locale) => {
     const fl = getCldrParentLocale(locale.toLowerCase())
-    if (fl) {
+    if (fl && !config.fallbackLocales[locale]) {
```

Comment:
What if one simply defines the default fallback locale and not each individual language's fallback?

Reply:
We tought that was a good idea to always build the cldr fallback and co-exist with default except if a user passes false to config. 

Anyways was a fast implementation and can be re-thinked

Reply:
Appreciated, the issue is that with 3.2.1, if you set `defaultLocales` to false and at the same time you add a local that is in the cldr list and the parent is not the sourceLocale, you still get errors down the road. So it would be nice if instead, if config has defaultLocales defined in my case with { default: 'en-US' } that this config not be overridden nor enhanced ... Anyways, it's my suggestion ... I'm grateful for all the hard work nonetheless, awesome library! Thanks again!